libraries/ESP32/examples/Camera/CameraWebServer/CameraWebServer.ino

```diff
@@ -62,6 +62,7 @@ void setup() {
     config.frame_size = FRAMESIZE_SVGA;
     config.jpeg_quality = 12;
     config.fb_count = 1;
+    config.fb_location = CAMERA_FB_IN_DRAM;
```

Comment:
this should be set to DRAM only if the board does not have PSRAM

Reply:
yes, that's why I added it into the else branch of  if(psramFound()). In hope that the psramFound() function returns false only if the board does not have PSRAM.

Reply:
ahh sorry I did not expand the code and did not see the if statement :)